src/apps/applib/lib/utility.c

```diff
@@ -163,3 +164,16 @@ int readBoundary(FILE* f, GeoBoundary* b) {
 
     return 0;
 }
+
+void assertExpectedBoundary(H3Index h3, const GeoBoundary* b1) {
```

Comment:
Moved this to `utility.c` to share - let me know if you think it should go into `test.c` instead.

Reply:
Is this used by non-tests? Seems like it should be in test.c.

Reply:
Agreed.